discover/faq.md

```diff
@@ -12,978 +12,157 @@ Hopefully, you'll find the answers here! If not then please don't hesitate to re
-PS: [Fine-grained permissions](../developers/fg-permissions.md) are not integrated into the Ocean Marketplace.
-
-</details>
-
-<details>
-
-<summary>Do other associated marketplaces like Acentrik, BigDataProtocol, and datalatte have any connection to the OCEAN token or Data Farming?</summary>
-
-Several projects use ocean components to power their data marketplaces. If these marketplaces are publicly accessible and indexable by Ocean Protocol, they are included in the data farming rewards program. If the marketplaces are closed and private, which cannot be indexed and tracked, then assets and activities are not part of the data farming program.
-
-</details>
-
-<details>
-
-<summary>Considering that the Ocean Market is open source and there are no incentives for industry players to use it, what are the future plans for the project in terms of its economic direction?</summary>
-
-The Ocean Market has consistently served as a showcase for the practical application of Ocean technology. Moreover, it has the potential to set a precedent for the development of other marketplaces within the Ocean ecosystem.
-
-However, it's important to note that participants using the Ocean stack are subject to transaction fees, which vary depending on the chosen token, either 0.1% or 0.2%. These fees serve the dual purpose of furthering the development of Ocean technology and facilitating the buy-and-burning of the Ocean token. 
-</details>
 
 <details>
+<summary>To what extent is Ocean becoming fully decentralized?</summary>
 
-<summary>Where can I find projects on Ocean's ecosystem?</summary>
+To be fully decentralized means no single point of control, at any level of the stack. The OCEAN token is already fully decentralized. The Ocean core tech stack is already fully decentralized too: smart contracts on permissionless chains, and anyone can run support middleware. The Data Farming incentives program has some centralized components; we aim to decentralize those in the next 12-24 months. ⁣
 
-All the ecosystem information is available on the [website](https://oceanprotocol.com/ecosystem).
+1. Details about the [Ocean Token](https://oceanprotocol.com/about-us/ocean-token).
+2. Details about the [Core Stack](../developers/contracts/README.md).
+3. Details about [Data Farming](../rewards/README.md).
   
 </details>
 
 <details>
+<summary>Where can we see previous data challenges and submitted solutions?</summary>
 
-<summary>Is there a real-time bot that tracks dataset sales on Ocean Protocol?</summary>
-
-Not to our knowledge. If you find something, please share it with us 🙃
-  
+You can find a list of past data challenges on the [website](https://oceanprotocol.com/challenges). 
+Accessing the submitted solutions is currently a work in progress, with one solution already available for the [Catalunya](https://catalunya.oceanprotocol.com/) challenge.
```

Comment:
Remove sentence "Accessing the submitted solutions".

Why: this will almost certainly get out of date quickly. And the info isn't _that_ useful here. So, remove it.